src/main/java/com/selfxdsd/todos/SelfTodosApplication.java

```diff
@@ -47,4 +55,27 @@ public static void main(final String[] args) {
         SpringApplication.run(SelfTodosApplication.class, args);
     }
 
+    /**
+     * Storage bean.
+     * @return Storage.
+     */
+    @Bean
+    @Scope(value = "prototype")
+    public Storage storage() {
+        return new SelfJooq(new MySql(
+            System.getenv(Env.DB_URL),
+            System.getenv(Env.DB_USER),
+            System.getenv(Env.DB_PASSWORD)
+        ));
+    }
+
+    /**
+     * PuzzlesReviewer bean.
+     * @return PuzzleReviewer.
+     * @throws IOException If something goes wrong.
+     */
+    @Bean
+    public PuzzlesReviewer puzzlesReviewer() throws IOException {
+        return new PuzzlesReviewer.DefaultPuzzleReviewer();
+    }
```

Comment:
we need these bean configuration to override it in tests... I expose the Storage as a higher abstraction instead of Database. Because SelfJooq is calling MySql#connect on constructor and String throw an error on test initialization.